### PR TITLE
fix: title 2 spec changed

### PR DESCRIPTION
**Pull request by liruchen32**
# Issue
[[會員制] 我的閱讀 - 新增及修改 design system 元件](https://app.asana.com/0/1203699264568808/1206252112911356/f) spec change

# Notice

# Dependency

### CODE REVIEW

File: packages/react-components/src/title-bar/stories/title2.stories.js [hunk 1]
```diff
@@ -18,15 +18,12 @@ export default {
...
-          <TextButton
-            text="RWD 按鈕"
-            rightIconComponent={<Arrow direction="right" />}
-            size={size}
-          />
-        )
-      : null
+    args.renderButton = args.showButton ? (
+      <TextButton
+        text="RWD 按鈕"
```
Comment (Aylie-Chou) [inline packages/react-components/src/title-bar/stories/title2.stories.js:23]: Could we modify the text if it's not a `RWD` button XD

**Review by Aylie-Chou — CHANGES_REQUESTED** [review]
Please have a look at comment ~

File: packages/react-components/src/title-bar/stories/title2.stories.js (hunk 1 above)
Comment (liruchen32) [reply packages/react-components/src/title-bar/stories/title2.stories.js:23]: fixed

**Review by Aylie-Chou — APPROVED** [review]
LGTM

**Pull request merged**